src/features/tags/components/TagManager/components/TagChip.tsx

```diff
@@ -100,17 +102,26 @@ const TagToolTip: React.FunctionComponent<{
 
 const TagChip: React.FunctionComponent<{
   disabled?: boolean;
+  noWrappedLabel?: boolean;
```

Comment:
Why does this need to be a property? Can't it always be `true`?

Reply:
Since tagChips are used in other components, I thought it would be better to give a property instead

Reply:
I understand why you would feel that way, and I think it's in many ways logical as a precaution when you make changes that you don't know make sense for all use cases. But this particular change seems like something that would make sense for all tags everywhere, so I would have made the change and then gone back to all other places where tags are used and make sure they still look right.